src/userscripts/linkHarmonyReleaseActions.user.js

```diff
@@ -0,0 +1,315 @@
+ */
+function processReleaseTable(tableBodySelector, digitalOnly) {
+  const releaseTableBody = document.querySelector(tableBodySelector);
+  if (!releaseTableBody) {
+    console.error(
+      `Harmony Link Script: Could not find release table body using selector: "${tableBodySelector}".`
+    );
+    return;
+  }
+
+  // Select BDI elements within release links in the second column
+  const releaseTitleBDIElements = releaseTableBody.querySelectorAll(
+    `tr:not(.subh) > td:nth-of-type(2) ${releaseTitleLinkSelector}`
+  );
+  let addedCount = 0;
+
+  releaseTitleBDIElements.forEach((bdiElement) => {
+    const releaseLink = bdiElement.parentElement;
+    if (!releaseLink || releaseLink.tagName !== "A") return;
+
+    const parentRow = releaseLink.closest("tr"); // Get the table row
+    if (!parentRow) return;
+
+    // --- Format Check ---
+    if (digitalOnly) {
+      const formatCell = parentRow.querySelector("td:nth-of-type(4)"); // 4th column is Format
+      const formatText = formatCell ? formatCell.textContent.trim() : "";
+      if (!formatText.includes("Digital Media")) {
+        // console.log("Skipping non-digital:", formatText);
+        return;
+      }
+      // console.log("Found digital:", formatText);
+    }
+
+    const linkEntity = extractEntityFromURL(releaseLink.href);
+
+    if (linkEntity && linkEntity.type === "release" && linkEntity.mbid) {
+      const mbid = linkEntity.mbid;
+      const harmonyLink = createHarmonyLinkElement(mbid);
+      const parentTd = releaseLink.closest("td"); // The containing cell (TD)
+
+      if (harmonyLink && parentTd) {
+        let nodeToInsertAfter = releaseLink;
+        // Traverse up the DOM tree from the release link
+        // until we find the element that is a DIRECT child of the TD.
+        // This handles cases where the link is wrapped in other elements (like span.mp).
+        while (nodeToInsertAfter.parentElement !== parentTd) {
+          nodeToInsertAfter = nodeToInsertAfter.parentElement;
+          if (
+            !nodeToInsertAfter ||
+            nodeToInsertAfter === parentTd ||
+            nodeToInsertAfter === document.body
+          ) {
+            console.error(
+              "Harmony Link Script: Could not find the correct insertion point within the TD for link:",
+              releaseLink.href
+            );
+            nodeToInsertAfter = null; // Prevent insertion if logic fails
+            break;
+          }
+        }
+
+        // Only insert if we successfully found the correct reference node
+        if (nodeToInsertAfter) {
+          try {
+            parentTd.insertBefore(harmonyLink, nodeToInsertAfter.nextSibling);
+            addedCount++;
+          } catch (e) {
+            console.error("Harmony Link Script: Error during insertion:", e);
+          }
+        }
+      }
+    }
+  });
+  // console.log(`Harmony Link Script: Processed table "${tableBodySelector}". Added ${addedCount} icons.`);
```

Comment:
Instead of commenting out these logger statements, you can use `console.debug`. The bundler will remove them automatically for production builds (`build` task) and only preserve them for debug builds (`debug-build` task).